Lib/glyphsLib/types.py

```diff
@@ -376,7 +376,8 @@ def encode_dict_as_string_for_gsnode(value):
     encoded such that it can be included in the string representation
     of a GSNode."""
     value = value.replace('"', '\\"')
-    return value.replace("\n", "\\n")
+    value = value.replace('\n', '')
+    return value.replace(";", ";\\n", value.count(';') - 1)
```

Comment:
What if one of the dict's values contain some `\n` or `;`? I think it's out of scope for this MR though, and will add more tests somewhere else.

Reply:
Yes, sorry, I don't like this userData thing either, but updating the Glyphs file to the latest version introduced that weirdness with newlines. Please make better! Thanks!